libs/designsystem/src/lib/components/chart/chart.component.html

```diff
@@ -1 +1 @@
-<div #chartContainer [style.height]="height + 'px'"></div>
+<div #chartContainer data-test="chart" [style.height]="height + 'px'"></div>
```

Comment:
Is this used anywhere? If not, delete:
```suggestion
<div #chartContainer [style.height]="height + 'px'"></div>
```